openebs/openebs-provisioner.go

```diff
@@ -90,7 +93,12 @@ func (p *openEBSProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 	volSize := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volumeSpec.Metadata.Labels.Storage = volSize.String()
 
-	volumeSpec.Metadata.Labels.StorageClass = *options.PVC.Spec.StorageClassName
+	className := GetPersistentVolumeClass(options)
+
+	if *className == "" {
```

Comment:
should there be a check for nil also here?

Reply:
className is string pointer , so thought was better to check the value of pointer

Reply:
done